### DISCUSSION TITLE
Error while installing on Ubuntu 22.4 VS Code

### DISCUSSION THREAD

**Discussion by sumyrai** [microsoft/vscode-cpptools#11589] [Q&A]
How to handle this 'Error while installing 'C/C++' extension. Please check the [log](command:workbench.action.showWindowLog) for more details'?

**Comment by sean-mcmanus** [comment]
@sumyrai VS Code is responsible for installing our extension, not our extension: https://github.com/microsoft/vscode/issues .

You could check the log for more details or restart VS Code and try again.
